doc/dev/mgmt/operation_migration.md

```diff
@@ -0,0 +1,135 @@
+
+**What will break**:
+
+- Code that passes positional parameters
+
+**Before**:
+
+```python
+from azure.mgmt.confluent import ConfluentManagementClient
+
+client = ConfluentManagementClient(credential, subscription_id)
+
+# Pass query parameters positionally
+environments = client.organization_operations.list_environments(
+    "resource_group",
+    "org_name", 
+    10,        # Pass page_size as positional
+    "token"    # Pass page_token as positional
+)
+```
+
+**After**:
+
+```python
+from azure.mgmt.confluent import ConfluentManagementClient
+
+client = ConfluentManagementClient(credential, subscription_id)
+
+# ❌ Raises TypeError
+environments = client.organization_operations.list_environments(
+    "resource_group",
+    "org_name", 
+    10,
+    "token"
+)
+
+# ✅ Use these approaches instead
+environments = client.organization_operations.list_environments(
+    "resource_group",
+    "org_name",
+    page_size=10,         # Must be passed with a keyword
+    page_token="token"    # Must be passed with a keyword
+)
+```
+
+**Migration steps:**
+
+- Convert all positional parameters to keyword arguments
+
+### Conditional Operation Parameters Changed
+
+**What changed**: Conditional operation headers `if_match` and `if_none_match` have been replaced with more semantic `etag` and `match_condition` parameters that use enum values to describe match conditions.
+
+**What will break**:
+
+- Code using `if_match` or `if_none_match` parameters
+
+**Before**:
+
+```python
+from azure.mgmt.containerservicefleet import ContainerServiceFleetManagementClient
+
+client = ContainerServiceFleetManagementClient(credential, subscription_id)
+
+# Update the resource only if the existing etag matches the value provided in this header
+fleet = client.fleets.begin_create_or_update(
+    resource_group_name="rg",
+    fleet_name="fleet1", 
+    resource=fleet_operation,
+    if_match=<specific etag>
+)
+```
+
+**After**:
+
+```python
+from azure.mgmt.containerservicefleet import ContainerServiceFleetManagementClient
+from azure.core import MatchConditions
+
+client = ContainerServiceFleetManagementClient(credential, subscription_id)
+
+# Use enum to describe match conditions
+fleet = client.fleets.begin_create_or_update(
+    resource_group_name="rg",
+    fleet_name="fleet1",
+    resource=fleet_operation,
+    etag=<specific etag>,
+    match_condition=MatchConditions.IfNotModified
+)
+```
+
+**MatchConditions enum values:**
+- `MatchConditions.IfNotModified` - Perform the operation only if the targeted resource's etag matches the specified etag
+- `MatchConditions.IfModified` - Perform the operation only if the targeted resource's etag does not match the specified etag
+- `MatchConditions.IfPresent` - Perform the operation only if the resource exists
+- `MatchConditions.IfMissing` - Perform the operation only if the resource does not exist
+- `MatchConditions.Unconditionally` - Perform the operation regardless of any condition
```

Comment:
We don't show an example about how to use this value so it is better to hide it.